newEvaluation.css

```diff
@@ -13,5 +13,91 @@ li
 
 fieldset
 {
-	max-width: 850px;
+	/*max-width: 850px;*/
+	width: 90%;
+}
+
+section
+{
+	width: 90%;
+}
+
+form
+{
+	flex-direction: column-reverse;
+}
+
+footer 
+{
+	position: fixed;
+	right: 0;
+	bottom: 0;
+	left: 0;
+	background-color: #efefef;
+	text-align: right;
+	color: #636262;
+	font-size: 10px;
+}
+
+footer p
+{
+	margin-right: 17%;
+}
+
+#submitButton
+{
+	background-color: #03A9F4;
+	color: #FFFFFF;
+	border: 0;
+	float:right;
+	padding: 7px 17px 7px 17px;
+	box-shadow: 0 4px 2px 1px rgba(182, 182, 182, 0.6);
+	border-radius: 3px;
+}
+
+legend, h5
+{
+	color: #03A9F4;
+}
+
+textarea
+{
+	resize: none;
+	height: 92.6%;
```

Comment:
No no, it doesn't :) I will try with px instead of percentage for width and I found that in this case I don't need height because i already have the rows property in html.